### ISSUE TITLE
webUi cannot display inceptionResnet Graph

### ISSUE THREAD

**Issue by vimalaguti** [deeplearning4j/deeplearning4j#4015]
#### Issue Description

Please describe our issue:
Trained the configuration of the model.InceptionResNetV1
seved the stats on a fileStatStorage file.
Once opened with the webUI and click on the model tab:
`[application-akka.actor.default-dispatcher-13] ERROR application - 

! @758oaf78p - Internal server error, for (GET) [/train/model/graph] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[InvocationTargetException: null]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:265) ~[play_2.11-2.4.6.jar:2.4.6]
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:191) ~[play_2.11-2.4.6.jar:2.4.6]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$9$$anonfun$apply$1.applyOrElse(PlayDefaultUpstreamHandler.scala:151) [play-netty-server_2.11-2.4.6.jar:2.4.6]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$9$$anonfun$apply$1.applyOrElse(PlayDefaultUpstreamHandler.scala:148) [play-netty-server_2.11-2.4.6.jar:2.4.6]
	at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:36) [scala-library-2.11.6.jar:na]
	at scala.util.Failure$$anonfun$recover$1.apply(Try.scala:215) [scala-library-2.11.6.jar:na]
	at scala.util.Try$.apply(Try.scala:191) [scala-library-2.11.6.jar:na]
	at scala.util.Failure.recover(Try.scala:215) [scala-library-2.11.6.jar:na]
	at scala.concurrent.Future$$anonfun$recover$1.apply(Future.scala:324) [scala-library-2.11.6.jar:na]
	at scala.concurrent.Future$$anonfun$recover$1.apply(Future.scala:324) [scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library-2.11.6.jar:na]
	at play.api.libs.iteratee.Execution$trampoline$.executeScheduled(Execution.scala:109) [play-iteratees_2.11-2.4.6.jar:2.4.6]
	at play.api.libs.iteratee.Execution$trampoline$.execute(Execution.scala:71) [play-iteratees_2.11-2.4.6.jar:2.4.6]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library-2.11.6.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:23) [scala-library-2.11.6.jar:na]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40) [akka-actor_2.11-2.3.13.jar:na]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:397) [akka-actor_2.11-2.3.13.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library-2.11.6.jar:na]
Caused by: java.lang.reflect.InvocationTargetException: null
	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_121]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_121]
	at play.routing.RouterBuilderHelper$$anonfun$build$1$$anonfun$apply$1$$anonfun$5.apply(RouterBuilderHelper.scala:66) ~[play-java_2.11-2.4.6.jar:2.4.6]
	at play.routing.RouterBuilderHelper$$anonfun$build$1$$anonfun$apply$1$$anonfun$5.apply(RouterBuilderHelper.scala:62) ~[play-java_2.11-2.4.6.jar:2.4.6]
	at play.api.mvc.Action$.invokeBlock(Action.scala:533) ~[play_2.11-2.4.6.jar:2.4.6]
	at play.api.mvc.Action$.invokeBlock(Action.scala:530) ~[play_2.11-2.4.6.jar:2.4.6]
	at play.api.mvc.ActionBuilder$$anon$1.apply(Action.scala:493) ~[play_2.11-2.4.6.jar:2.4.6]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5.apply(Action.scala:105) ~[play_2.11-2.4.6.jar:2.4.6]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5.apply(Action.scala:105) ~[play_2.11-2.4.6.jar:2.4.6]
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21) ~[play_2.11-2.4.6.jar:2.4.6]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4.apply(Action.scala:104) ~[play_2.11-2.4.6.jar:2.4.6]
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4.apply(Action.scala:103) ~[play_2.11-2.4.6.jar:2.4.6]
	at scala.Option.map(Option.scala:146) ~[scala-library-2.11.6.jar:na]
	at play.api.mvc.Action$$anonfun$apply$1.apply(Action.scala:103) ~[play_2.11-2.4.6.jar:2.4.6]
	at play.api.mvc.Action$$anonfun$apply$1.apply(Action.scala:96) ~[play_2.11-2.4.6.jar:2.4.6]
	at play.api.libs.iteratee.Iteratee$$anonfun$mapM$1.apply(Iteratee.scala:524) ~[play-iteratees_2.11-2.4.6.jar:2.4.6]
	at play.api.libs.iteratee.Iteratee$$anonfun$mapM$1.apply(Iteratee.scala:524) ~[play-iteratees_2.11-2.4.6.jar:2.4.6]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1.apply(Iteratee.scala:560) ~[play-iteratees_2.11-2.4.6.jar:2.4.6]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1.apply(Iteratee.scala:560) ~[play-iteratees_2.11-2.4.6.jar:2.4.6]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$13.apply(Iteratee.scala:536) ~[play-iteratees_2.11-2.4.6.jar:2.4.6]
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$13.apply(Iteratee.scala:536) ~[play-iteratees_2.11-2.4.6.jar:2.4.6]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) [scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) [scala-library-2.11.6.jar:na]
	... 6 common frames omitted
Caused by: java.lang.RuntimeException: org.nd4j.shade.jackson.databind.JsonMappingException: Could not resolve type id 'CenterLossOutputLayer' into a subtype of [simple type, class org.deeplearning4j.nn.conf.layers.Layer]: known type ids = [FrozenLayer, GlobalPooling, LSTM, Layer, RBM, VariationalAutoencoder, activation, autoEncoder, batchNormalization, convolution, convolution1d, dense, dropout, embedding, gravesBidirectionalLSTM, gravesLSTM, localResponseNormalization, loss, output, rnnoutput, subsampling, subsampling1d, zeroPadding]
 at [Source: java.io.StringReader@61f2bad9; line: 25109, column: 13] (through reference chain: org.deeplearning4j.nn.conf.ComputationGraphConfiguration["vertices"]->java.util.LinkedHashMap["outputLayer"]->org.deeplearning4j.nn.conf.graph.LayerVertex["layerConf"]->org.deeplearning4j.nn.conf.NeuralNetConfiguration["layer"])
	at org.deeplearning4j.nn.conf.ComputationGraphConfiguration.fromJson(ComputationGraphConfiguration.java:160) ~[deeplearning4j-nn-0.9.1.jar:na]
	at org.deeplearning4j.ui.module.train.TrainModule.getConfig(TrainModule.java:633) ~[deeplearning4j-play_2.11-0.9.1.jar:na]
	at org.deeplearning4j.ui.module.train.TrainModule.getGraphInfo(TrainModule.java:601) ~[deeplearning4j-play_2.11-0.9.1.jar:na]
	at org.deeplearning4j.ui.module.train.TrainModule.getModelGraph(TrainModule.java:594) ~[deeplearning4j-play_2.11-0.9.1.jar:na]
	... 30 common frames omitted
Caused by: org.nd4j.shade.jackson.databind.JsonMappingException: Could not resolve type id 'CenterLossOutputLayer' into a subtype of [simple type, class org.deeplearning4j.nn.conf.layers.Layer]: known type ids = [FrozenLayer, GlobalPooling, LSTM, Layer, RBM, VariationalAutoencoder, activation, autoEncoder, batchNormalization, convolution, convolution1d, dense, dropout, embedding, gravesBidirectionalLSTM, gravesLSTM, localResponseNormalization, loss, output, rnnoutput, subsampling, subsampling1d, zeroPadding]
 at [Source: java.io.StringReader@61f2bad9; line: 25109, column: 13] (through reference chain: org.deeplearning4j.nn.conf.ComputationGraphConfiguration["vertices"]->java.util.LinkedHashMap["outputLayer"]->org.deeplearning4j.nn.conf.graph.LayerVertex["layerConf"]->org.deeplearning4j.nn.conf.NeuralNetConfiguration["layer"])
	at org.nd4j.shade.jackson.databind.JsonMappingException.from(JsonMappingException.java:148) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.DeserializationContext.unknownTypeException(DeserializationContext.java:948) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.jsontype.impl.TypeDeserializerBase._handleUnknownTypeId(TypeDeserializerBase.java:275) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.jsontype.impl.TypeDeserializerBase._findDeserializer(TypeDeserializerBase.java:162) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.jsontype.impl.AsWrapperTypeDeserializer._deserialize(AsWrapperTypeDeserializer.java:100) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.jsontype.impl.AsWrapperTypeDeserializer.deserializeTypedFromObject(AsWrapperTypeDeserializer.java:49) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.AbstractDeserializer.deserializeWithType(AbstractDeserializer.java:131) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:521) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:95) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.impl.BeanPropertyMap.findDeserializeAndSet(BeanPropertyMap.java:285) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:248) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:136) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:523) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:95) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.impl.BeanPropertyMap.findDeserializeAndSet(BeanPropertyMap.java:285) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:248) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:136) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.jsontype.impl.AsWrapperTypeDeserializer._deserialize(AsWrapperTypeDeserializer.java:114) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.jsontype.impl.AsWrapperTypeDeserializer.deserializeTypedFromObject(AsWrapperTypeDeserializer.java:49) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.AbstractDeserializer.deserializeWithType(AbstractDeserializer.java:131) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.std.MapDeserializer._readAndBindStringMap(MapDeserializer.java:475) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.std.MapDeserializer.deserialize(MapDeserializer.java:330) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.std.MapDeserializer.deserialize(MapDeserializer.java:25) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:523) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:95) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.impl.BeanPropertyMap.findDeserializeAndSet(BeanPropertyMap.java:285) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:248) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:136) ~[jackson-0.9.1.jar:na]
	at org.deeplearning4j.nn.conf.serde.ComputationGraphConfigurationDeserializer.deserialize(ComputationGraphConfigurationDeserializer.java:28) ~[deeplearning4j-nn-0.9.1.jar:na]
	at org.deeplearning4j.nn.conf.serde.ComputationGraphConfigurationDeserializer.deserialize(ComputationGraphConfigurationDeserializer.java:18) ~[deeplearning4j-nn-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:3562) ~[jackson-0.9.1.jar:na]
	at org.nd4j.shade.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2578) ~[jackson-0.9.1.jar:na]
	at org.deeplearning4j.nn.conf.ComputationGraphConfiguration.fromJson(ComputationGraphConfiguration.java:158) ~[deeplearning4j-nn-0.9.1.jar:na]
	... 33 common frames omitted`

#### Version Information

Please indicate relevant versions, including, if relevant:

* Deeplearning4j version 0.9.1
* platform information (OS, etc) rhel
* CUDA version, if used 8
cudnn 6
* NVIDIA driver version, if in use

**Comment by AlexDBlack** [comment]
> Caused by: java.lang.RuntimeException: org.nd4j.shade.jackson.databind.JsonMappingException: Could not resolve type id 'CenterLossOutputLayer' into a subtype of [simple type, class org.deeplearning4j.nn.conf.layers.Layer]

That's the cause, though normally it shouldn't be an issue, as the layer will be found via reflections.
https://github.com/deeplearning4j/deeplearning4j/issues/4533

The fact that it doesn't is a little odd, and I'm not sure why.
Do you get any reflections-related logs/errors/warnings earlier in the program execution?

A possible workaround here, before running the UI:
```
import org.nd4j.shade.jackson.databind.jsontype.NamedType;
List<NamedType> types = new ArrayList<>();
types.add(new NamedType(CenterLossOutputLayel.class, "CenterLossOutputLayel"));
NeuralNetConfiguration.reinitMapperWithSubtypes(types);
```

**Comment by agibsonccc** [comment]
Closing — references a removed/rewritten subsystem (DataVec, NLP/Word2Vec, Model Zoo, Spark, UI, or Arbiter). These have been fundamentally rewritten or removed in the upcoming release. Please open a new issue if you experience similar problems with the new version.
